src/App.tsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
 
 import './App.scss';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+const preparedPosts: Post[] = [];
+
+posts.forEach((post) => {
```

Comment:
Можно использовать метод map
```suggestion
const preparedPosts = posts.map((post) => {
```